app/assets/stylesheets/common/base/onebox.scss

```diff
@@ -480,6 +480,11 @@ pre.onebox code {
   .github-info-container {
     width: 100%;
     overflow-x: hidden;
+    > span {
+      // legacy, can be removed in the future
```

Comment:
would be better to use TODO (kris) and maybe a date where you expectit to be removable